src/main/java/hu/oe/nik/szfmv/automatedcar/model/Road.java

```diff
@@ -7,7 +7,6 @@
 import javax.xml.bind.Unmarshaller;
 import java.awt.Shape;
 import java.awt.Rectangle;
-import java.awt.Polygon;
 import java.awt.geom.*;
```

Comment:
Majd #151 -ben "refaktorálom" (nekem automata megcsinálja az IDEA a packageket a beállításaim szerint szóval elég csak ránéznem a fájlra és automata megoldja a *-ot)